Src/VimCore/CoreInterfaces.fs

```diff
@@ -4491,17 +4509,23 @@ and IMarkMap =
     /// Get the mark for the given char for the IVimTextBuffer
     abstract GetMark: mark: Mark -> vimBufferData: IVimBufferData -> VirtualSnapshotPoint option
 
+    /// Get the mark info for the given char for the IVimTextBuffer
+    abstract GetMarkInfo: mark: Mark -> vimBufferData: IVimBufferData -> (char * string * int * int) option

```

Comment:
Should we add a small record here vs returning a 4 way tuple?

Reply:
I created a `MarkInfo` type which `GetMarkInfo` now returns.